windows/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM microsoft/nanoserver:sac2016
+FROM mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
- For the Docker image name change: it should be done in the file `tmpl.Dockerfile` instead of here
- For the tag (version) change, it should be done in the scripts `update*.sh` as they generate these Dockerfiles from the templates.